### ISSUE TITLE
Use the dropdown-divider class on divider items when useBootstrap4 is true

### ISSUE THREAD

**Issue by willroman** [isaacplmann/angular2-contextmenu#38]
Would it be possible to add Bootstrap 4's dropdown-divider class on divider context menu items?  I having been trying to get a divider item to show with Bootstrap 4 by defining the divider class in my own CSS file (with the same settings as dropdown-divider) but I'm not having any luck getting it to actually style the entry.  When I modified the JavaScript directly to apply the dropdown-divider class, it began working as expected.

In examining the source code, contextMenu.component.ts, a change like the following (starting on line 38) would seem to do the trick:

FROM
        `<li *ngFor="let menuItem of visibleMenuItems" [class.disabled]="!isMenuItemEnabled(menuItem)"
          [attr.role]="menuItem.divider ? 'separator' : undefined" [class.divider]="menuItem.divider">`

TO
        `<li *ngFor="let menuItem of visibleMenuItems" [class.disabled]="!isMenuItemEnabled(menuItem)" 
         [class.divider]="menuItem.divider" [class.dropdown-divider]="useBootstrap4 && menuItem.divider"
         [attr.role]="menuItem.divider ? 'separator' : undefined">`

Thanks in advance for your consideration.

**Comment by isaacplmann** [comment]
Yes! I don't actually use bootstrap 4 do it's great to get feedback from
people that do. Why don't you submit a pr since you've already written the
code? That way you get credit as a contributor.

**Comment by willroman** [comment]
Thanks for the opportunity.  Unfortunately, I work in a Windows/Visual Studio environment and, even though I was able to successfully fork the code, I can't seem to get the build scripts to work so I can't really verify that everything will be correct.  I've even been trying things from within VS Code without any luck.  I can still make the change and submit the PR if not being able to verify things myself is alright with you.

As for using Bootstrap 4, it was something we chose to go with since we are also adopting Angular 2 and working on some major re-writes of some of our internal software.  It seems to be in pretty good shape at this point but, with the number of changes it has from Bootstrap 3, I can see why a lot of people are going to take their time in moving to it.

I do have to say that I like the way you built in a switch into your code so that it can be really easy to deal with the Bootstrap 4 differences when they come up.

Bill
